definitions/npm/express_v4.x.x/flow_v0.25.x-/express_v4.x.x.js

```diff
@@ -12,7 +12,7 @@ declare module 'express' {
   }
   declare class Request extends http$IncomingMessage mixins RequestResponseBase {
     baseUrl: string;
-    body: mixed;
+    body: any;
```

Comment:
> you can't assign an `Array` to an `Object`

This was an intentional design decision that trades off between utility and exactly modeling JS. It's easy to make an argument in both directions really, so we went with the more conservative expectation (that an "Array" is disjoint from the concept of an "Object": Objects have named properties, arrays are usually treated more vector-y).

Pedantically: Anything that you can do with an `Object` could also be done with an `Array`, but practically this is probably not often the intention.